tests/test_reader_scenarios.py

```
@@ -181,9 +190,17 @@ def test_reader_graph_navigation(temp_vault):
-        st, obj = _get(port, obj_links[0])
+        marker = "id='cluster-graph-data'>"
+        start = graph.index(marker) + len(marker)
+        end = graph.index("</script>", start)
+        data = json.loads(graph[start:end].replace("<\\/", "</"))
+        obj_paths = [
+            node["path"]
+            for node in data["nodes"]
+            if isinstance(node.get("path"), str) and node["path"].startswith("/object?")
+        ]
+        assert obj_paths, "/map JSON data should carry at least one /object? path"
+        st, obj = _get(port, obj_paths[0])
```

Ruff (0.15.12)

[warning] Unpacked variable `obj` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)